src/tests/test_parse.py

```diff
@@ -505,3 +506,29 @@ def test_escape_string_slicing_notation(query, expected_escaped, expected_found)
     escaped, found = escape_string_slicing_notation(query)
     assert escaped == expected_escaped
     assert found == expected_found
+
+
+@pytest.mark.parametrize(
+    "line, expected",
+    [
+        (
+            '%sql select "{"a": "b"}"::json -> "a"',
```

Comment:
Please add some multiline tests also:
```
select '[1,2,3]'::json 
->>
2
```

Reply:
Added [multiline](https://github.com/ploomber/jupysql/pull/938/files#diff-edd543175e806ac6654196270596c90240cc3a3cfdcba20d35d997207e4555f0R530-R560) cases with different variations of spaces/tabs/newlines